insteon_mqtt/device/KeypadLinc.py

```diff
@@ -870,6 +870,41 @@ def set_on_level(self, level, on_done=None):
 
         self.send(msg, msg_handler)
 
+    #-----------------------------------------------------------------------
+    def set_ramp_rate(self, rate, on_done=None):
+        """Set the device default ramp rate.
+
+        This changes the dimmer default ramp rate of how quickly the it
+        will turn on or off. This rate can be between .1 seconds and up
+        to 9 minutes.
+
+        Args:
+          rate (int): The default ramp rate in the range [0,31]
```

Comment:
Sorry for the late comment.  I noticed that this code is expecting a ramp rate value that would be handed directly to the device -- so the user would be expected to know what value to use based on how those values are interpreted by devices (such as by consulting [this table](http://www.madreporite.com/insteon/ramprate.htm).)

I just wanted to raise the question - would it be better if the ramp rate were specified in seconds, and translated using the self.ramp_pretty lookup table, similar to as done in link_data_from_pretty()?

This might be a question for @krkeegan or @TD22057 to answer, though.